pkg/controller/daemon/util/daemonset_util.go

```diff
@@ -115,3 +118,48 @@ func SplitByAvailablePods(minReadySeconds int32, pods []*v1.Pod) ([]*v1.Pod, []*
 	}
 	return availablePods, unavailablePods
 }
+
+// AppendDaemonSetPodNodeAffinity creates NodeAffinity entry for DaemonSet controller
```

Comment:
The comment is a bit confusing. Thinking about this function alone, I think something like the following would be more appropriate:
// AppendDaemonSetPodNodeAffinity adds a NodeAffinity term with the given "nodeName" to the "affinity" terms.

Reply:
done